tox.ini

```
@@ -1,4 +1,30 @@
+[testenv]
+deps = .[test]
+commands = nose2 -c tox.ini
+
 [flake8]
 ignore = E501, E402
-exclude = .git,__pycache__,docs/source/conf.py,old,build,dist
-max-complexity = 10
\ No newline at end of file
+exclude = .git,__pycache__,docs/source/conf.py,old,build,dist,venv,.env,.eggs,.tox
+max-complexity = 10
+# The GitHub editor is 127 chars wide
+max-line-length = 127 
```

I'd recommend using 99 max because anything much longer will wrap when doing side-by-side diffs (when merging, etc).

```suggestion
max-line-length = 99
```